### ISSUE TITLE
Wrist pose head detection issue

### ISSUE THREAD

**Issue by Ropelopunny** [deviant-dev/ds-issues#11]
Describe the bug:
  The game does not recognize that the head is in the correct position during present wrist poses in both head Mistress and Nega Yuki scenes.

To Reproduce:
   Attempting to do the present wrist to HeadMistress or nega yuki.

Expected behavior : 
  The game recognizing the head is in the right position when doing the present wrist pose for the Head Mistress and Nega Yuki scenes

Screenshots:
   Present wrists from Head Mistress Scene - Feet flat on ground
![20200201021145_1](https://user-images.githubusercontent.com/60497088/73588934-58e24700-449d-11ea-9613-11e066071e91.jpg)

   Present wrists from Head Mistress Scene - Standing on toes
![20200201021149_1](https://user-images.githubusercontent.com/60497088/73588954-7c0cf680-449d-11ea-9e7c-7e6d503dbb97.jpg)

   Raised Chin pose from head Mistress Scene - Feet flat on ground
![20200201021213_1](https://user-images.githubusercontent.com/60497088/73588957-8b8c3f80-449d-11ea-99c2-b406b328fd52.jpg)

  Present wrist reward pose from nega yuki scene - feet flat on ground
![20200201022241_1](https://user-images.githubusercontent.com/60497088/73588968-aeb6ef00-449d-11ea-957e-9b530ba65cf8.jpg)

  Present wrist reward pose from nega yuki scene - standing on toes
![20200201022245_1](https://user-images.githubusercontent.com/60497088/73588974-b9718400-449d-11ea-881e-fd1638934362.jpg)

  Raised chin pose from nega yuki scene - feet flat on ground
![20200201022142_1](https://user-images.githubusercontent.com/60497088/73588976-be363800-449d-11ea-8823-411e3fd10ef7.jpg)

System (please complete the following information):
* VR Device: Index HMD, controllers, and 2.0 lighthouse
* Version: Build 50

Additional context:
 - Player hight: ~5’ 2” or ~1.58m
 - Issue was not present in build 44
 - Standing poses work without issue

**Comment by Ropelopunny** [comment]
An update on this issue, this may be tied to the issues with the floor in the head Mistress Scene. Includes are some screenshots of the scenes outside the head Mistress scene where the floor is where it should be and screenshots of the headmistress scene where both characters and the player is hovering above where the floor should be. Screenshots presented is from build 73

![20200615114754_1](https://user-images.githubusercontent.com/60497088/84692139-63b0ec00-af13-11ea-8032-52aa51176052.jpg)

![20200615131027_1](https://user-images.githubusercontent.com/60497088/84692170-72979e80-af13-11ea-862c-1ba2c0de8506.jpg)

**Comment by Ropelopunny** [comment]
Update: After further testing with the current Steam build 1.1.0 I don't think the issue is tied to the floor height issue in Head Mistress scene. The floor height issue will be will be brought into its own issue. The issue seems to be tied with standing poses in general, with the headset showing at the neck of the example model on standing poses as opposed to kneeling poses where it shows where it should be. If possible a calibration where the game calibrates the position of the headset when standing and/or kneeling before starting the game, providing your height to the game for it to make the adjustments. Included are shots of standing and kneeling positions.

![20200619131611_1](https://user-images.githubusercontent.com/60497088/85164180-f910f080-b231-11ea-9ee6-beee0d275f05.jpg)

![20200619132549_1](https://user-images.githubusercontent.com/60497088/85164207-04fcb280-b232-11ea-9954-76ead41982e8.jpg)

![20200619132621_1](https://user-images.githubusercontent.com/60497088/85164238-0fb74780-b232-11ea-8031-11e2e97e7b74.jpg)

**Comment by Ropelopunny** [comment]
As of latest build 81 wrist detection during present stones poses seems to have been resolved

**Comment by deviant-dev** [comment]
Thanks for following up on this, @Ropelopunny! I'll mark it as closed, then.

**Comment by deviant-dev** [comment]
From Ropey:
> The present wrist pose is better I no longer have to get a step ladder or stand on my toes to do it but I do need to tilt my head up higher than I need to do on other stand poses.

**Comment by Ropelopunny** [comment]
Link of head position needed during normal pose and how far back head needs to be tilted for HM scene

https://www.dropbox.com/sh/0yezv7ugx9q9j0d/AAC1NtaG1q9v6nwuIOTewuYRa?dl=0

**Comment by Ropelopunny** [comment]
Issue confirmed resolved for both poses in build 2.0.1. Closing issue
